mash_client/cli/config.py

```diff
@@ -119,7 +119,59 @@ def setup_config(
     config_dir, profile, email, host, port, log_level, color, verify
 ):
     """
-    Creates a client config file based on user input to prompts.
+    Create a configuration file for the mash command line tool
+
+    .SH OPTIONS
+    .TP
+    \\fB--color\\fP Colorize the output
+    By default the output is not colorized. Using \\fI--color\\fP as an option
+    will enable output colorization.
+    .TP
+    \\fB--config-dir\\fP PATH_TO_CONFIGURATION_DIR
```

Comment:
This option is mainly for development and very specific use cases. It should not really be used by an end user and it is not stored in the configuration file which would be recursive.

We could hide the option if we want but it's very helpful for dev/testing.